### PR TITLE
XXX Temporary branch XXX

### PR DESCRIPTION
**Pull request by jayz25**

### CODE REVIEW

File: pages/typeahead.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        setDisplayedUsers(getUserswithMatchingName(textInputValue));
+    }
+
+    const onKeyUpInput = (event: React.KeyboardEvent) => {
+        event.preventDefault();
+        if (event.key === "Enter") {
+            toggleSuggestionsDropdown();
+        }
+        if (!textInput.current.value && event.key === "Backspace") {
+            const selected = document.querySelector(".selected-users-container .selected-user-item:last-of-type");
```
Comment (ashwinkhode) [inline pages/typeahead.tsx:89]: dont do this ever

File: pages/typeahead.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+    const onSelectedItemKeyUp = (e: React.KeyboardEvent, user: User) => {
+        if (e.key === "Backspace" && document.activeElement === (e.target)) {
+            onDeselectItem(user);
+            const selected = document.querySelector(".selected-users-container .selected-user-item:nth-last-of-type(2)");
+            if (selected) {
+                (selected as HTMLElement)?.focus();
+            } else {
+                (document.querySelector(".user-typeahead") as HTMLElement).focus();
+            }
```
Comment (ashwinkhode) [inline pages/typeahead.tsx:102]: no need for all this black magic fuckery

File: pages/typeahead.tsx (hunk 1 above)
Comment (jayz25) [reply pages/typeahead.tsx:89]: @ashwinkhode Thanks for review, will update to use Refs here.

File: pages/typeahead.tsx (hunk 2 above)
Comment (jayz25) [reply pages/typeahead.tsx:102]: @ashwinkhode Okay, thanks.

**Pull request closed without merging**